File: Go/5/5_CalculoIMC.go
package main

import (
	"fmt"
)

func calcularIMC(peso, altura float64) string {
	imc := peso / (altura * altura)

	if imc < 18.5 {
		return "Abaixo do peso"
	} else if imc >= 18.5 && imc < 25 {
		return "Peso normal"
	} else if imc >= 25 && imc < 30 {
		return "Sobrepeso"
	} else if imc >= 30 && imc < 35 {
		return "Obesidade de nível 1"
	} else {
		return "Obesidade de nível 2"
	}
}

func main() {
	peso := 70.0   // em kg
	altura := 1.75 // em metros

	resultadoIMC := calcularIMC(peso, altura)
	fmt.Println("O seu IMC é:", resultadoIMC)
}

File: Js/5_CalculoIMC.js
function imc(peso, altura) {
    const total = peso / (altura * altura)

    if (total <= 18.5) {
        return "Seu IMC é de : " + total + " Nível 0 -> - MAGREZA"
    } else if (total > 18.5 && total <= 24.9) {
        return "Seu IMC é de : " + total + " Nível 0 -> - NORMAL"
    } else if (total > 24.9 && total <= 29.9) {
        return "Seu IMC é de : " + total + " Nível I -> SOBREPESO"
    } else if (total > 29.9 && total <= 39.9) {
        return "Seu IMC é de : " + total + " Nível II -> OBESIDADE"
    } else (total > 40) 
    return "Seu IMC é de : " + total + " Nível III -> OBESIDADE GRAVE"
}

console.log(imc(49,1.54))